### ISSUE TITLE
DecodingFailure when running validate/standalone

### ISSUE THREAD

**Issue by Enoch2090** [metarank/metarank#956]
Hi! I'm trying to replicate the hybrid search article that Vsevolod posted with Pinecone: [Hybrid Search and Learning-to-Rank with Metarank](https://www.pinecone.io/learn/metarank/). I manually created data with one of my projects. Here are the files that I used:
- [config.yml](https://gist.github.com/Enoch2090/9c9836e8fe0b763177c6bf76c9349908#file-config-yml)
- [events.jsonl](https://gist.github.com/Enoch2090/414c18b709ab1fdc87016105190fe005#file-events-jsonl)

I used Metarank 0.6.1, the command is
```bash
java -jar metarank-0.6.1.jar standalone --config config.yml --data events.jsonl
```

Which returns 
```bash
22:16:13.898 INFO  ai.metarank.main.Main$ - Metarank v0.6.1 is starting.
DecodingFailure(cannot parse a feature definition of type 'number', List(DownField(source), DownArray, DownField(features)))
```
Not sure if I constructed any of the files wrongly...?
p.s. The cos feature in my events.jsonl is actually dot product similarity. But since that is just a variable name I didn't change it.

**Comment by vgoloviznin** [comment]
Hey @Enoch2090, one weirdness that I see is that your events file doesn't contain item metadata events: https://docs.metarank.ai/reference/event-schema#item-metadata-event. I'm not sure if this is connected to the bug you see though, we will take a look at it!

**Comment by shuttie** [comment]
So looks like that we had a mismatch in config syntax between our docs and the actually expected one. The [doc](https://docs.metarank.ai/reference/overview/feature-extractors/scalar) (and your example config.yml) says that number feature is configured like this:
```yaml
- name: price
  type: number
  field: item.price // must be a number
  scope: item
```
But the code expects the following:
```yaml
- name: price
  type: number
  source: item.price // must be a number
  scope: item
```

With the fix in https://github.com/metarank/metarank/pull/960 Metarank now accepts both syntax. 

Can you try the [0.6.4](https://github.com/metarank/metarank/releases/tag/0.6.4) which contains the fix for this issue? With 0.6.4 when using your `config.yml` and `events.jsonl` files it now seems to work OK.

**Comment by Enoch2090** [comment]
Thanks for the quick fix! It works perfectly now.
